pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -161,7 +162,9 @@ stdenv.mkDerivation rec {
   ) ++ lib.optionals stdenv.isLinux [
     bubblewrap
     libseccomp
+  ] ++ lib.optionals systemdSupport [
     systemd
+  ] ++ lib.optionals stdenv.isLinux [
```

Comment:
not needed, instead move the dependencies to the above isLinux

btw add your commit email to your github account so the commits get associated with your account

Reply:
Okay, done.  I was trying to minimize rebuilds by keeping the derivation the same in the default case, but you're right long-term maintainability matters more.